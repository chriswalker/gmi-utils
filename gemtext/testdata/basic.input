# Sample header

A long line of text that should get split up properly by the formatter correctly. It is required that clients ndle line breaks and pagination correctly, and that raw gemtext files do not contain artificial line splits.

This is another line.

> This is a quote

## Sample subheader

* Bullet item 1
* Bullet item 2

### Three-level subheader

=> Gemini Link gemini://some.url/
