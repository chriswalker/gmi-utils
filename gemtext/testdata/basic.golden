  [38;2;0;255;255m# Sample header[39m
  
  A long line of text that
  should get split up
  properly by the formatter
  correctly. It is required
  that clients ndle line
  breaks and pagination
  correctly, and that raw
  gemtext files do not
  contain artificial line
  splits.
  
  This is another line.
  
  [38;2;255;0;255m> This is a quote[39m
  
  [38;2;0;255;255m## Sample subheader[39m
  
  * Bullet item 1
  * Bullet item 2
  
  [38;2;255;255;0m### Three-level subheader[39m
  
  [38;2;0;255;0mLink gemini://some.url/ [Gemini][39m
